recipes/docker.rb

```diff
@@ -17,9 +17,11 @@
 # limitations under the License.
 #
 
-docker_service 'default' do
-  #tls false
-  #provider Chef::Provider::DockerService::Sysvinit
-  action [:create, :start]
-  group 'wheel'
+if node['os_version'] !~ /^3\.2\.0.*$/
```

Comment:
Is this necessary for RHEL, CentOS and Ubuntu support? If so, maybe add comment explaining purpose. Also recommend using `unless` for readability.